src/app/dataset-view/additional-components/dataset-settings-component/tabs/scheduling/dataset-settings-scheduling-tab.component.ts

```diff
@@ -73,22 +62,52 @@ export class DatasetSettingsSchedulingTabComponent extends BaseComponent {
             .subscribe();
     }
 
-    public savePollingTriggers(pollingForm: FormGroup<PollingGroupType>): void {
+    public changePollingTriggers(pollingForm: FormGroup<PollingGroupType>): void {
+        this.pollingForm = pollingForm;
+    }
+
+    public saveScheduledUpdates(): void {
         this.datasetSchedulingService
-            .setDatasetTriggers({
+            .setDatasetFlowConfigs({
                 datasetId: this.datasetBasics.id,
                 datasetFlowType: DatasetFlowType.Ingest,
-                paused: !pollingForm.controls.updatesState.value,
-                triggerInput: this.setPollingTriggerInput(pollingForm),
-                datasetInfo: {
-                    accountName: this.datasetBasics.owner.accountName,
-                    datasetName: this.datasetBasics.name,
+                configInput: {
+                    ingest: {
+                        fetchUncacheable: this.ingestConfigurationForm.controls.fetchUncacheable.value,
+                    },
                 },
             })
-            .pipe(takeUntilDestroyed(this.destroyRef))
+            .pipe(
+                switchMap((success) => {
+                    if (success) {
+                        return this.datasetSchedulingService.setDatasetTriggers({
+                            datasetId: this.datasetBasics.id,
+                            datasetFlowType: DatasetFlowType.Ingest,
+                            paused: !this.pollingForm.controls.updatesState.value,
+                            triggerInput: this.setPollingTriggerInput(this.pollingForm),
+                            datasetInfo: {
+                                accountName: this.datasetBasics.owner.accountName,
+                                datasetName: this.datasetBasics.name,
+                            },
+                        });
+                    } else {
+                        this.toastrService.error("Configuration is not saved");
+                        return EMPTY;
+                    }
```

Comment:
Where is a similar handling when trigger is not saved?

Reply:
Error handling is in the method itself when handling triggers.